zwave_js_server/model/endpoint.py

```diff
@@ -64,3 +67,75 @@ def installer_icon(self) -> Optional[int]:
     def user_icon(self) -> Optional[int]:
         """Return user icon property."""
         return self.data.get("userIcon")
+
+    async def _async_send_command(
+        self,
+        cmd: str,
+        require_schema: Optional[int] = None,
+        wait_for_result: Optional[bool] = None,
+        **cmd_kwargs: Any,
+    ) -> Optional[Dict[str, Any]]:
+        """
+        Send an endpoint command. For internal use only.
+
+        If wait_for_result is not None, it will take precedence, otherwise we will decide to wait
+        or not based on the node status.
+        """
+        if self.client.driver is None:
+            raise FailedCommand(
+                "Command failed", "failed_command", "The client is not connected"
+            )
+        node = self.client.driver.controller.nodes[self.node_id]
+        kwargs = {}
+        message = {
+            "command": f"endpoint.{cmd}",
+            "nodeId": self.node_id,
+            "endpoint": self.index,
+            **cmd_kwargs,
+        }
+        if require_schema is not None:
+            kwargs["require_schema"] = require_schema
+
+        if wait_for_result or (
+            wait_for_result is None and node.status != NodeStatus.ASLEEP
+        ):
+            result = await self.client.async_send_command(message, **kwargs)
+            return result
+
+        await self.client.async_send_command_no_wait(message, **kwargs)
+        return None
+
+    async def async_invoke_cc_api(
+        self,
+        command_class: CommandClass,
+        method_name: str,
+        *args: Any,
+        wait_for_result: bool = None,
+    ) -> Any:
+        """Call endpoint.invoke_cc_api command."""
+        result = await Endpoint._async_send_command(
```

Comment:
```suggestion
        result = await self._async_send_command(
```

Reply:
The reason I forced this to be `Endpoint` is because if, for whatever reason, the `Node` picks up `_async_send_command`, the command will fail (it always has to be `endpoint.invoke_cc_api` and `endpoint.supports_cc_api`). Alternatively, we can update zwave-js-server to accept `node.invoke_cc_api`.

Reply:
I don't find the code that is limiting this?

Reply:
it doesn't right now, but actually my tests will catch this in case it does change in the future, so we will go with the suggested approach